examples/async-sqlite/README.md

```diff
@@ -37,7 +37,7 @@ Rust data cannot be directly held by JavaScript. The [`JsBox`][jsbox] provides a
 
 #### Channels and `EventQueue`
 
-The mpsc channel provides a way for the JavaScript main thread to communicate with the database thread, but it is one-way. In order to complete the callback, the database thread must be able to communicate with the JavaScript main thread. [`EventQueue`][eventqueue] provides a channel for sending these events back.
+The mpsc channel provides a way for the JavaScript main thread to communicate with the database thread, but it is one-way. In order to complete the callback, the database thread must be able to communicate with the JavaScript main thread. [`Channel`][channel] provides a channel for sending these events back.
```

Comment:
It might be good to scope `Channel` here to make it clear it's a Neon channel.

```suggestion
The mpsc channel provides a way for the JavaScript main thread to communicate with the database thread, but it is one-way. In order to complete the callback, the database thread must be able to communicate with the JavaScript main thread. [`neon::event::Channel`][channel] provides a channel for sending these events back.
```